boards/arm/mps4/doc/index.rst

```diff
@@ -0,0 +1,188 @@
+.. zephyr:board:: mps4
+
+ARM MPS4
+###############
```

Comment:
this actually needs to go away :) zephyr board directive takes care of this already :)

```suggestion
```